jetcd-launcher/src/main/java/io/etcd/jetcd/launcher/Etcd.java

```diff
@@ -48,6 +50,13 @@ public static class Builder {
         private Network network;
         private boolean shouldMountDataDirectory = true;
 
+        public Builder(){
+            String image = System.getenv("ETCD_IMAGE");
+            if (!Strings.isNullOrEmpty(image)) {
+                withImage(image);
```

Comment:
invoking a non final method from the constructor is usually a bad practice as could lead to unexpected results, better to set the instance variable directly.